packages/server-wallet/src/__test__/create-and-close-channel/direct-funding.test.ts

```diff
@@ -152,25 +152,27 @@ it('Create a directly funded channel between two wallets ', async () => {
     status: 'running',
     turnNum: 3,
   });
-});
 
-it('Rejects b closing with `not your turn`', async () => {
```

Comment:
Combining these three tests into one, as they only work if run sequentially in order anyway.